fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -1184,10 +1184,34 @@ public boolean shouldTriggeredRefreshBy(String dbName, String tableName) {
             return true;
         }
         List<TableName> excludedTriggerTables = tableProperty.getExcludedTriggerTables();
-        if (excludedTriggerTables == null) {
+        return matchTable(excludedTriggerTables, dbName, tableName);
+    }
+
+    public boolean shouldRefreshBaseTable(String tableName) {
```

Comment:
Add some comments, since excluded refresh tables may cause mv's inconsistence with mv's definition query which is not expected in some situations.

Reply:
I will add documentation when this PR is approved.

https://github.com/StarRocks/starrocks/blob/main/docs/en/sql-reference/sql-statements/materialized_view/CREATE_MATERIALIZED_VIEW.md?plain=1#L268